src/ocean/io/FileSystem.d

```diff
@@ -53,139 +46,124 @@ version (Posix)
 
 struct FileSystem
 {
-        /***********************************************************************
```

Comment:
Could you separate all the indentation fixes? :grimacing:

Reply:
I hope you meant "indentation fixes not in version" blocks !
Sure, will do.